src/common/utility.h

```diff
@@ -54,18 +54,35 @@ namespace Utility {
     OCSYNC_EXPORT void usleep(int usec);
     OCSYNC_EXPORT QString formatFingerprint(const QByteArray &, bool colonSeparated = true);
     /**
-     * @brief Creates the Desktop.ini file which contains the folder IconResource shown as a favorite link
+     * @brief Create favorite link for sync folder with application name and icon
      *
      * @param folder absolute file path to folder
      */
     OCSYNC_EXPORT void setupFavLink(const QString &folder);
+    /**
+     * @brief Migrate favorite link for sync folder with new application name and icon
+     *
+     * @param folder absolute file path to folder
+     * @param lnkName the new name of the lnk file
+     */
+    OCSYNC_EXPORT void migrateFavLink(const QString &folder, const QString &lnkName = QString());
+    /**
+     * @brief Creates or overwrite the Desktop.ini file to use new folder IconResource shown as a favorite link
+     *
+     * @param folder absolute file path to folder
+     * @param overwrite boolean to create or overwrite ini file
+     */
+    OCSYNC_EXPORT void setupDesktopIni(const QString &folder, bool overwrite = false);
```

Comment:
usually a `bool`is not the best idea in such APIs
when you call `setupDesktopIni(path, true);` what this `true` means ?
can you use an enum ?